### DISCUSSION TITLE
Adopt shadcn/ui 😎

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#224] [Announcements]
I'm excited to announce that we've adopted [shadcn/ui](https://ui.shadcn.com/) in The Epic Stack 🎉

This means it will be much easier for folks to take advantage of that great registry of components and the components that come shipped with the Epic Stack will be better thought through (especially with regard to light/dark mode support).

Enjoy the decision document here: https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/019-components.md

The commit for this is quite large as this is a pretty impactful change, but hopefully it helps anyone else who would like to make this same change in their own Epic Stack app: https://github.com/epicweb-dev/epic-stack/commit/bc93804353cf89c8c901e1a8b629ad25ac0a4e3c

As a reminder, you should not feel compelled to adopt this yourself. Only do so if you feel like your app needs it. Your app will still work just as well today as it did yesterday :) But if you decide to adopt this same change, hopefully the commit will be helpful.

**Comment by DonKoko** [comment]
This is awesome. I have been using it for quite some time and I think its the perfect mix between having flexibility and getting something that already looks polished. 
Love to see it.

**Comment by andrecasal** [comment]
@kentcdodds, thank you for integrating shadcn/ui into the Epic Stack, this is excellent 🎉
